zerver/views/user_groups.py

```diff
@@ -138,12 +138,20 @@ def is_api_value_same_as_current_setting_value(
     ) == set(current_value["direct_subgroups"])
 
 
-def check_setting_value_changed(
+def can_edit_group_setting(
```

Comment:
Here again, this can be named better.

Reply:
Maybe `validate_group_setting_value_change`